go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534
 	github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
 	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
-	github.com/Microsoft/hcsshim v0.0.0-20190417211021-672e52e9209d
+	github.com/Microsoft/hcsshim v0.8.7
```

Comment:
This delta introduces the `k8s.io/kubernetes` transitive dependency (and master still depends on k8s.io/cri-api). It is an oversight that our verify scripts do not catch this. Opened https://github.com/kubernetes/kubernetes/pull/89531 to fix that.

Before we can pick this up, I think the following has to happen:
* we need to cut k8s.io/cri-api loose so it is not a staging repo
* hcsshim needs to cut another tag (or we can use a SHA from master)

Reply:
please see https://github.com/microsoft/hcsshim/pull/783 if they cut a release with that ... is that enough?

Reply:
That certainly narrows the scope... it would mean we'd have a `k8s.io/kubernetes -> vendor/microsoft/hcsshim -> k8s.io/cri-api` in our dependency tree... not sure what `hack/verify-no-vendor-cycles.sh` would think of that. It might not notice since we don't actually link to the hcsshim packages that depend on cri-api.

I tried updating to hcsshim master and got this error, so there may be additional steps required to pull in that version:
```
=== tidying k8s.io/client-go
k8s.io/client-go/plugin/pkg/client/auth/azure imports
	github.com/Azure/go-autorest/autorest: ambiguous import: found github.com/Azure/go-autorest/autorest in multiple modules:
	github.com/Azure/go-autorest v10.8.1+incompatible (/Users/liggitt/go/pkg/mod/github.com/!azure/go-autorest@v10.8.1+incompatible/autorest)
	github.com/Azure/go-autorest/autorest v0.9.6 (/Users/liggitt/go/pkg/mod/github.com/!azure/go-autorest/autorest@v0.9.6)
k8s.io/client-go/plugin/pkg/client/auth/azure imports
	github.com/Azure/go-autorest/autorest/adal: ambiguous import: found github.com/Azure/go-autorest/autorest/adal in multiple modules:
	github.com/Azure/go-autorest v10.8.1+incompatible (/Users/liggitt/go/pkg/mod/github.com/!azure/go-autorest@v10.8.1+incompatible/autorest/adal)
	github.com/Azure/go-autorest/autorest/adal v0.8.2 (/Users/liggitt/go/pkg/mod/github.com/!azure/go-autorest/autorest/adal@v0.8.2)
k8s.io/client-go/plugin/pkg/client/auth/azure imports
	github.com/Azure/go-autorest/autorest/azure: ambiguous import: found github.com/Azure/go-autorest/autorest/azure in multiple modules:
	github.com/Azure/go-autorest v10.8.1+incompatible (/Users/liggitt/go/pkg/mod/github.com/!azure/go-autorest@v10.8.1+incompatible/autorest/azure)
	github.com/Azure/go-autorest/autorest v0.9.6 (/Users/liggitt/go/pkg/mod/github.com/!azure/go-autorest/autorest@v0.9.6/azure)
```

Reply:
I would recommend getting pinning to master of hcsshim working with the other azure dependencies before asking them to tag a release. If we can get that happy and passing our existing vendor cycle checks, that seems a workable step forward as we work through getting cri-api standalone

Reply:
sounds like a plan!